query/src/main/java/org/infinispan/query/cq/CQResult.java

```diff
@@ -0,0 +1,70 @@
+package org.infinispan.query.cq;
+
+import org.infinispan.commons.marshall.AbstractExternalizer;
+import org.infinispan.query.impl.externalizers.ExternalizerIds;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author anistor@redhat.com
+ * @since 8.0
+ */
+public final class CQResult<V> {
+
```

Comment:
Not a big deal, by the shorthand "CQ" for  "Continuous Query" always confuses my with "Clustered Queries" since it's being used by clustered query in some class names  :smile:  

Could you think of another naming? I'm afraid it will affect this PR in several places...